app/models/response_map.rb

```
@@ -59,31 +87,88 @@ def aggregate_reviewers_score
+      submitted_found = true
       response_score += response.aggregate_questionnaire_score
       total_score += response.maximum_score
     end
 
-    # If no submitted responses at all, return nil
     return nil unless submitted_found
 
-    # Return the normalized score (as a float), or 0 if no valid total score
-    total_score > 0 ? (response_score.to_f / total_score) : 0
+    total_score.positive? ? (response_score.to_f / total_score) : 0
+  end
```

RuboCop (1.86.1)

[convention] Cyclomatic complexity for `aggregate_reviewers_score` is too high. [9/7]

(Metrics/CyclomaticComplexity)